r2r/examples/clients/run_basic_client.py

```diff
@@ -49,6 +51,14 @@ def search(self, query):
             print(body[:500])
             print("\n")
 
+    def rag_chatbot(self, query, model="gpt4-turbo-preview"):
```

Comment:
We might want to make a new python file called 'run_chatbot_client' and then rename "basic" to "qna" or some other shorthand for "question and answer".

Much of the "basic" pipeline logic revolves around Q&A and it might be nice to make this distinction.